### PR TITLE
Store form template as jsonb on the side of old relational way

### PR DESCRIPTION
**Pull request by hukka**

### CODE REVIEW

File: resources/sql/queries.sql [hunk 1]
```diff
@@ -207,6 +207,36 @@ WHERE 1=1
...
+SELECT
+  id,
+  organization,
+  title,
+  start,
+  endt as "end",
+  fields::TEXT,
+  enabled,
+  archived,
+  TRUE AS active -- TODO implement active and archiving
```
Comment (luontola) [inline resources/sql/queries.sql]: `rems.db.core/assoc-active` can be used to calculate active.

File: resources/sql/queries.sql [hunk 2]
```diff
@@ -207,6 +207,36 @@ WHERE 1=1
...
+FROM form_template
+WHERE id = :id;
+
+-- :name get-form-templates
+select id from form_template;
```
Comment (luontola) [inline resources/sql/queries.sql]: Dead code?

File: test/clj/rems/api/test_forms.clj [hunk 3]
```diff
@@ -78,10 +78,20 @@
...
+                                  assert-response-is-ok
+                                  read-body)]
             (is (= 1 (count forms))
                 "only one form got created")
             ;; TODO: create an API for reading full forms (will be needed latest for editing forms)
+            (testing "form template matches command"
+              (is (= (select-keys command [:title :organization])
+                     (select-keys form-template [:title :organization])))
+              (is (= (:items command)
+                     (:fields form-template))))
```
Comment (luontola) [inline test/clj/rems/api/test_forms.clj]: Also the command's `:items` should be renamed to `:fields`. It can also be done in a different PR if it seems too big a change.

File: resources/sql/queries.sql (hunk 1 above)
Comment (hukka) [reply resources/sql/queries.sql]: Hm, I copy pasted from other queries. I have to see if I will fix the other parts here or in other PR.

File: resources/sql/queries.sql (hunk 2 above)
Comment (hukka) [reply resources/sql/queries.sql]: Yeah… We need a proper query to replace the old plural query eventually, but this is not it. I'll remove it.

File: test/clj/rems/api/test_forms.clj (hunk 3 above)
Comment (hukka) [reply test/clj/rems/api/test_forms.clj]: Didn't want to touch the UI code yet. Will do later.

**Comment by opqdonut** [comment]
for #835
